airflow/providers/apache/hive/hooks/hive.py

```diff
@@ -426,7 +426,7 @@ def load_file(
         if create or recreate:
             if field_dict is None:
                 raise ValueError("Must provide a field dict when creating a table")
-            fields = ",\n    ".join(['`{k}` {v}'.format(k=k.strip('`'), v=v) for k, v in field_dict.items()])
+            fields = ",\n    ".join([f"`{k.strip('`')}` {v}" for k, v in field_dict.items()])
```

Comment:
This change looks unrelated -- is it?

Reply:
Hi @ashb Yes, the changes in hive.py were unrelated but the test scripts failed on this preventing me to open the PR. I needed to fix there... don't know why :-(

Reply:
It _should_ be ignored if you don't otherwise change the file.